procedures/release-duty/desktop/esr-to-esr-updates.rst

```diff
@@ -83,3 +83,35 @@ have been superceded by the new ``esr*`` rules.
 .. |platform-deprecation| image:: /procedures/release-duty/desktop/platform-deprecation.png
 .. |watershed| image:: /procedures/release-duty/desktop/watershed.png
 
+
+The Snap case
+-------------
+
+In addition to Balrog, you need to update the Snap store too. For more information about Snap,
+see `procedures/release-duty/desktop/ubuntu-snap.rst`. Steps:
+
+1. `docker pull snapcore/snapcraft:stable`
+2. `docker run -ti snapcore/snapcraft:stable`
+3. `snapcraft login`
```

Comment:
Nit, not blocking: Maybe add a quick note that the credentials are in the private repo?